test/integration/host_js_test.rb

```diff
@@ -225,7 +225,7 @@ class HostJSTest < IntegrationTestWithJavascript
       page.find('#check_all').click
 
       # Ensure all hosts are checked
-      assert page.find('input.host_select_boxes').checked?
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
```

Comment:
Ah thanks, I wasn't aware of those selectors.  The double negative is unfortunately slightly important - it verifies that all of the checkboxes are selected (none of them are unchecked). If changed to has_checked_field? then it'd return true if there's one or more checkboxes checked, which is slightly different.

I think the equivalent would be `assert page.has_no_unchecked_field?('input.host_select_boxes')`.